### PR TITLE
Df len

### PR DESCRIPTION
**Pull request by Rubtsowa**
Method worked is not correct. Always returned result lambda was 0.

### CODE REVIEW

File: sdc/tests/test_dataframe.py [hunk 1]
```diff
@@ -1879,6 +1879,15 @@ def test_impl():
...
         self.assertTrue(isinstance(three, np.ndarray))
 
+    def test_df_len(self):
+        def test_impl(df):
+            return len(df)
+
+        hpat_func = self.jit(test_impl)
+        n = 11
+        df = pd.DataFrame({'A': np.arange(n), 'B': np.arange(n) ** 2})
+        assert (hpat_func(df), test_impl(df))
```
Comment (AlexanderKalistratov) [inline sdc/tests/test_dataframe.py]: This assert is always true.
Why do you use plain assert at all?

Comment (Rubtsowa) [reply sdc/tests/test_dataframe.py]: because result len() - number.

Comment (AlexanderKalistratov) [reply sdc/tests/test_dataframe.py]: https://docs.python.org/3/library/unittest.html#assert-methods

**Comment by AlexanderKalistratov** [comment]
too many files and changes

**Pull request merged**